cohortextractor/emis_backend.py

```diff
@@ -488,7 +488,7 @@ def get_temp_table_name(self, suffix):
         table_name += suffix
         return self.add_table_prefix(table_name)
 
-    def get_date_condition(self, table, date_expr, between):
+    def get_date_condition(self, table, date_expr, between, base_join_table=None):
```

Comment:
I may well have missed something, but looking through this function I can't see where else `table` is used other than the place we're now using `base_join_table`. So do we actually need the extra argument? Or do we just need to change the value we're supply as `table` when we call it?

Reply:
I think you're right - I missed that that was the only place it was used (and I was originally trying to do something much more unnecessarily complicated just for the ONS Deaths table)